pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/JdbiUmbrellaStudy.java

```diff
@@ -52,9 +52,9 @@
             + " from umbrella_study as us"
             + " join umbrella as u on us.umbrella_id = u.umbrella_id"
             + " left join olc_precision op on op.olc_precision_id = us.olc_precision_id"
-            + " where u.umbrella_name = :umbrella")
```

Comment:
This is the only spot that needed a change- looking up the umbrella using the GUID instead of it's human readable name.